homeassistant/components/media_player/directv.py

```diff
@@ -54,67 +58,241 @@
 def setup_platform(hass, config, add_entities, discovery_info=None):
     """Set up the DirecTV platform."""
     known_devices = hass.data.get(DATA_DIRECTV, set())
-    hosts = []
+    directv_entity = None
 
+    discovered = False
     if CONF_HOST in config:
+        name = config.get(CONF_NAME)
```

Comment:
Use [] syntax for anything that is required or an optional with a default.

Example: `name = config[CONF_NAME]`